source/frontend/src/components/common/form/ContactInput/ContactInputView.tsx

```diff
@@ -109,30 +119,44 @@ const ContactInputView: React.FunctionComponent<IContactInputViewProps> = ({
 
 export default ContactInputView;
 
-const StyledDiv = styled.div`
-  background: none;
+const StyledInputLikeDiv = styled.div<{ $hasContact: boolean }>`
```

Comment:
I'm not sure I get this name, isn't this just a div with a classname pointing to a styled components class? why is this "like" a div?

Reply:
I think it makes it clear that that div is imitiating the look of an input.